pkg/configuration/config.go

```diff
@@ -73,7 +73,7 @@ func Load(version string) (*Config, error) {
 	}
 	defer userConfigFile.Close()
 
-	var userConf *UsersConfig
+	var userConf *UsersConfig = &UsersConfig{}
 	err = loadConfig(userConfigFile, userConf)
```

Comment:
```suggestion
	err = loadConfig(userConfigFile, &userConf)
```
`loadConfig` 함수의 파라미터를 이렇게 수정해보시겠어요?

Reply:
제가 잘못 알고 있었네요. 포인터 변수라도 Unmarshal할 때는 &를 붙이는 게 맞고 이상 없네요.